### ISSUE TITLE
Update on deep learning embeddings

### ISSUE THREAD

**Issue by goncinious** [jump-cellpainting/datasets#100]
Are there are plans to release an updated set of deep learning profiles, perhaps obtained with [DeepProfiler](https://github.com/cytomining/DeepProfiler)?

**Comment by shntnu** [comment]
> Are there are plans to release an updated set of deep learning profiles, perhaps obtained with [DeepProfiler](https://github.com/cytomining/DeepProfiler)?

Yes, using `Cell Painting CNN v1` https://zenodo.org/records/7114558; this is underway

**Comment by shntnu** [comment]
This is now available, e.g., https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0016-jump/source_4/workspace_dl/profiles/cpcnn_zenodo_7114558/

Also, assembled versions are available via https://github.com/jump-cellpainting/2024_Chandrasekaran_Production/blob/c8c52550c14b95a1d8b6b51f7fe07fdab7728141/manifests/profile_index.json; we might added these to the manifest in this repo but we are keeping them separate for the moment

More on these manifests here https://broadinstitute.github.io/jump_hub/howto/maintenance/1_project_manifest.html
